playwright/tests/index.ts

```diff
@@ -0,0 +1,62 @@
+import { test as base } from '@playwright/test'
+import { mkdtemp, writeFile } from 'node:fs/promises'
+import path from 'path'
+import childProcess from 'child_process'
+import { tmpdir } from 'node:os'
+import serveStatic = require('serve-static')
+import connect = require('connect')
+import * as http from 'node:http'
+
+async function buildApp (projectDirectoryPath: string, bettyConfiguration: object): Promise<string> {
+  // We do not know the real base URL, but as Betty requires one, set an obviously fake value.
```

Comment:
We know the base URL is 127.0.0.1, so hardcode that

Reply:
But we don't know the port yet... So we'll have to start the server, get the port, then generate the site